### PR TITLE
Use unix domain socket by default

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel mrunalp@gmail.com

### CODE REVIEW

**Comment by mrunalp** [comment]
@runcom @hmeng-19 PTAL

**Comment by runcom** [comment]
👍

File: cmd/server/main.go [hunk 1]
```diff
@@ -3,18 +3,23 @@ package main
...
 
 const (
-	port = ":49999"
+	unixDomainSocket = "/var/run/ocid.sock"
 )
 
 func main() {
-	lis, err := net.Listen("tcp", port)
+	// Remove the socket if it already exists
+	if err := os.Remove(unixDomainSocket); err != nil {
```
Comment (haiyanmeng) [inline cmd/server/main.go]: @mrunalp , should we first check whether `unixDomainSocket` exists before `os.Remove` it?

Comment (mrunalp) [inline cmd/server/main.go]: good catch, pushed fix.

**Comment by haiyanmeng** [comment]
@mrunalp , I tested it, and it works perfectly.

**Comment by mrunalp** [comment]
@hmeng-19 Thanks for testing. Not sure why build is failing on git validation.

**Comment by mrunalp** [comment]
Merging as the git-validation passes for me locally.

**Pull request merged**